.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "Bug report"
+about: "Report a bug in Inferno"
+---
+
+Thank you for reporting a possible bug in Inferno! Please fill in as much of the template below as you can.
+
+Subject of the issue
```

Comment:
This file should have some formatting

Reply:
It should, in fact. Not sure what bit of copy-pasting I did got rid of it.